HOTT-1381 Show banner news items

jebw opened the pull request:
### Jira link

[HOTT-1381](https://transformuk.atlassian.net/browse/HOTT-1381)

### What?

I have added/removed/altered:

- [x] Added querying the latest banner from NewsItem
- [x] Added caching of that query
- [x] Show the banner news item in the header banner
- [x] Changed the govspeak sanitizer to allow `<a target="_blank"` links

### Why?

I am doing this because:

- We want to drive the header banner from the news items, controlled via the admin area.

### Deployment risks (optional)

- Affects every page, should not be merged before the backend changes are released

jebw committed:
HOTT-1381 Added retrieval of banner news items

jebw committed:
HOTT-1381 Cache the latest banner

jebw committed:
HOTT-1381 Drive banner partials from news items

jebw committed:
HOTT-1381 Allow target attribute on <a> links

alexdesi commented on app/models/news_item.rb:
```diff
@@ -53,6 +64,12 @@ def any_updates?
       end
     end
 
+    def cached_banner
```
Looking where this is called, 
it looks it does not matter if it is cached or not,
I guess we could call this just `last_banner` (or maybe, even just `banner`)

alexdesi commented on config/initializers/govspeak_sanitizer.rb:
```diff
@@ -0,0 +1,16 @@
+  end
+end
+
+Govspeak::HtmlSanitizer.prepend TariffGovspeakSanitizer
```
monkey patching is a last resort, wondering if we cannot just replace that piece of content (not very elegant, but maybe less bad than monkey patching (and easier to understand).

jebw committed:
HOTT-1381 Bump the cache key version for NewsItems

The data model we are caching is changing, so match the cache key to the version of the model being cached

alexdesi reviewed (approved):
It looks good to me✅
I've discussed some potential improvements (not blocking).

jebw committed:
HOTT-1381 Addressed review comments
